guides/common/modules/proc_overriding-ansible-variables-in-project.adoc

```diff
@@ -6,6 +6,11 @@ If you run Ansible roles in {Project}, you can use {Project} to override Ansible
 The following procedure refers to hosts and host groups.
 For more information, see {ManagingHostsDocURL}[_{ManagingHostsDocTitle}_].
 
+[WARNING]
+====
+To be able to use overriden Ansible variables, a user must have a role that allows them to see the those variables.
```

Comment:
```suggestion
To be able to use overridden Ansible variables, a user must have a role that allows them to see those variables.
```
Some options I prepared for you if you like:
1) Users must have roles allowing them to see overridden Ansible variables in order to use them.
2) The user must have the right role to see overridden Ansible variables if they want to use them.

Reply:
Thank you for the fix, applied.